osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -111,7 +111,7 @@ private void load(SessionStatics sessionStatics)
                     Children = new PlayerSettingsGroup[]
                     {
                         VisualSettings = new VisualSettings(),
-                        new InputSettings()
+                        new InputSettings(),new AudioSettings()
```

Comment:
```suggestion
                        new InputSettings(),
                        new AudioSettings()
```

Reply:
Please do not mark conversations as "resolved" while they're actually not.